### PR TITLE
Add support for ntp.conf so we can change time servers.

### PR DESCRIPTION
**Pull request by blockchainunchained**

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
Thanks for the PR. To make it really configurable I suggest to move it to the template folder and expose a variable with a list of ntp servers or something like that

**Comment by blockchainunchained** [comment]
I've not run this yet as I'm away from my dev machine.

**Comment by MaximilianMeister** [comment]
Looks good to be merged, only one small thing can you still add the new vars to the readme?

**Comment by blockchainunchained** [comment]
Yup will do, having a bit of a day off today will get on this tomorrow. I'll rebase if you make any changes upstream.

**Comment by blockchainunchained** [comment]
This is all squashed and should be ready to merge.

File: templates/ntp.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+# If you want to provide time to your local subnet, change the next line.
+# (Again, the address is an example only.)
+#broadcast 192.168.123.255
+
+# If you want to listen to time broadcasts on your local subnet, de-comment the
+# next lines.  Please do this only if you trust everybody on the network!
+#disable auth
+#broadcastclient
```
Comment (blockchainunchained) [inline templates/ntp.conf.j2:63]: Should there be a blank line at the end of this file?

Comment (MaximilianMeister) [reply templates/ntp.conf.j2:63]: probably not an issue

**Comment by MaximilianMeister** [comment]
> This is all squashed and should be ready to merge.

done, thanks :+1:

**Pull request merged**